sphinx_qt_documentation/utils.py

```diff
@@ -51,7 +51,9 @@ def _get_signal_and_version():
 
 
 Signal, QT_VERSION = _get_signal_and_version()
-
+SIGNAL_PREFIXES_REGEX = "".join(
+    rf"(?:{name}\.)?" for name in Signal.__module__.split(".")
+)
```

Comment:
As I understand, this makes each part of the module path optional. So the final regexp will match:

1. `Signal()`
2. `QtCore.Signal()`
3. `PyQt5.QtCore.Signal()` 

But also `PyQt5.Signal()`. But maybe it is acceptable. If you do not have idea how to fix it, I will try to do this, or just merge and release this PR.

Reply:
Yep that's exactly what it does. Maybe I can change the regex prefix so that it can't match PyQt5.Signal.

But maybe we should consider it reasonable as it would imply that it is not valid code in the first place.

Let me know which option you prefer.